src/Maker/MakeAuthenticator.php

```diff
@@ -381,11 +382,11 @@ private function generateNextMessage(bool $securityYamlUpdated, string $authenti
     private function userClassHasEncoder(array $securityData, string $userClass): bool
     {
         $userNeedsEncoder = false;
-        if (isset($securityData['security']['encoders']) && $securityData['security']['encoders']) {
-            foreach ($securityData['security']['encoders'] as $userClassWithEncoder => $encoder) {
-                if ($userClass === $userClassWithEncoder || is_subclass_of($userClass, $userClassWithEncoder)) {
-                    $userNeedsEncoder = true;
-                }
+        $hashersData = $securityData['security']['encoders'] ?? $securityData['security']['encoders'] ?? [];
+
+        foreach ($hashersData as $userClassWithEncoder => $encoder) {
+            if ($userClass === $userClassWithEncoder || is_subclass_of($userClass, $userClassWithEncoder) || class_implements($userClass, $userClassWithEncoder)) {
+                $userNeedsEncoder = true;
```

Comment:
This fixes an unrelated bug, where we weren't checking the newer "hashers" and we weren't accounting for an interface to be used there.